.circleci/config.yml

```
@@ -48,7 +48,17 @@ jobs:
             - ./node_modules
       - run: npm run build
       - run: npm run test
-
+  test-environment:
+    docker:
+      - image: cimg/base:2021.10
+    working_directory: ~/mozilla/glam/
+    steps:
+      - checkout
+      - run:
+          command: | # bind {PWD} because we use it in docker-compose.yml
+            cp .env-dist .env
+            make up -v ${PWD}:/app
+            make build -v ${PWD}:/app
```

I would skip the `make up` for now (unless/until we have a test that validates that the full environment comes up as expected)

```suggestion
           make build -v ${PWD}:/app
```